data/authorization_plugins.yml

```diff
@@ -21,4 +21,13 @@
   author_name: ThoughtWorks Inc.
   contact_url: mailto:support@thoughtworks.com?subject=Info about LDAP/AD Authentication Plugin
   paid: true
+  
+- name: Atlassian Crowd Authentication Plugin
+  description: Delegates authentication services to Atlasian Crowd
+  readmore_url: https://gitlab.com/mgambini1975/go-crowd-plugin.git
+  author_url: http://www.ditechonline.it/
+  author_name: Di.Tech S.p.A.
+  contact_url: mailto:marco.gambini@ditechspa.it?subject=Info about Crowd Authentication Plugin
+  paid: false
```

Comment:
There should be a releases_url. The idea is that users can just download the jar in server plugins external. Check the other yaml files. The reason the authorization plugins don't have the releases url is that its already bundled with go and users don't have to download it.